### PR TITLE
[RI1] Conversion to rst

### LINKED ISSUE #2758: Remove all duplicates in title (e.g. Introduction and Synopsis)
We need the autosectionlabel [1] support for sphinx to manipulate labels as expected by CNTT
See the ref to RA-2 Chapter 4.9 in https://raw.githubusercontent.com/cntt-n/CNTT/master/doc/ref_cert/RC2/chapters/chapter04.rst

This currently doesn't work because of multiple duplicates in titles in the same RST
RC1 and RC2 are already wrong here. But it should occur in all stream.
All title must be unique in the same (maxdepth=2).

[1] https://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html

```cedric@dev:~/Devs/OPNFV/CNTT/doc/ref_cert/RC2$ LANG=C tox
docs installed: alabaster==0.7.12,Babel==2.9.1,certifi==2021.5.30,charset-normalizer==2.0.4,doc8==0.9.0,docutils==0.17.1,idna==3.2,imagesize==1.2.0,Jinja2==3.0.1,MarkupSafe==2.0.1,packaging==21.0,pbr==5.6.0,Pygments==2.10.0,pyparsing==2.4.7,pytz==2021.1,requests==2.26.0,restructuredtext-lint==1.3.2,snowballstemmer==2.1.0,Sphinx==4.1.2,sphinx-rtd-theme==1.0.0,sphinxcontrib-applehelp==1.0.2,sphinxcontrib-devhelp==1.0.2,sphinxcontrib-htmlhelp==2.0.0,sphinxcontrib-jsmath==1.0.1,sphinxcontrib-qthelp==1.0.3,sphinxcontrib-serializinghtml==1.1.5,stevedore==3.4.0,urllib3==1.26.6
docs run-test-pre: PYTHONHASHSEED='3393966718'
docs run-test: commands[0] | doc8 . --ignore-path .tox --ignore-path build
Scanning...
Validating...
========
Total files scanned = 8
Total files ignored = 135
Total accumulated errors = 0
Detailed error counts:
    - doc8.checks.CheckCarriageReturn = 0
    - doc8.checks.CheckIndentationNoTab = 0
    - doc8.checks.CheckMaxLineLength = 0
    - doc8.checks.CheckNewlineEndOfFile = 0
    - doc8.checks.CheckTrailingWhitespace = 0
    - doc8.checks.CheckValidity = 0
docs run-test: commands[1] | sphinx-build -W -b html . build
Running Sphinx v4.1.2
loading intersphinx inventory from https://cntt.readthedocs.io/en/latest/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 7 source files that are out of date
updating environment: [new config] 7 added, 0 changed, 0 removed
reading sources... [ 14%] chapters/chapter01
reading sources... [ 28%] chapters/chapter02
reading sources... [ 42%] chapters/chapter03
reading sources... [ 57%] chapters/chapter04
reading sources... [ 71%] chapters/chapter05
reading sources... [ 85%] chapters/chapter06
reading sources... [100%] index


Warning, treated as error:
/home/cedric/Devs/OPNFV/CNTT/doc/ref_cert/RC2/chapters/chapter01.rst:43:duplicate label chapters/chapter01:introduction, other instance in /home/cedric/Devs/OPNFV/CNTT/doc/ref_cert/RC2/chapters/chapter01.rst
ERROR: InvocationError for command /home/cedric/Devs/OPNFV/CNTT/doc/ref_cert/RC2/.tox/docs/bin/sphinx-build -W -b html . build (exited with code 2)
_____________________________________________________________________________________________________ summary _____________________________________________________________________________________________________
ERROR:   docs: commands failed```

### LINKED ISSUE #2778: [RI1] Conversion from markdown to rst
Conversion from markdown to rst

### PR DESCRIPTION
**Pull request by gkunz**
This PR converts the RI1 documentation from markdown to rst.

Fixes #2778 
Fixes #2758

### CODE REVIEW

File: doc/ref_impl/cntt-ri/chapters/chapter03.rst [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+
+Consumable Infrastructure Resources and Services
+
++-------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+| Requirement Area                                | Description                                                                                                                                       |
++=================================================+===================================================================================================================================================+
+| Support for Profiles and T-shirt instance types | should follow tabels specified in `RA1: 4.4.1 Support for Profiles and T-shirt instance                                                           |
+|                                                 | types <ref_arch/openstack/chapters/chapter04#4.4.1>`                                                                                              |
```
Comment (CsatariGergely) [inline doc/ref_impl/cntt-ri/chapters/chapter03.rst:410]:
```suggestion
|                                                 | types <../../../ref_arch/openstack/chapters/chapter04.md#4.4.1>`__                                                                                |```

File: doc/ref_impl/cntt-ri/chapters/chapter06.rst [hunk 2]
```diff
@@ -0,0 +1,563 @@
...
+======================
+
+Introduction
+------------
+
+**must**: Requirements that are marked as *must* are considered mandatory and must exist in the reference implementation and implemented by installer. The same applies to *must not*.
+
+**may**: Requirements that are marked as *may* are considered optional. The same applies to *may not*.
+
+.. _installer-requirements-1:
```
Comment (CsatariGergely) [inline doc/ref_impl/cntt-ri/chapters/chapter06.rst:11]:
```suggestion
.. _ri1-ch6-requirements:
```

File: doc/ref_impl/cntt-ri/chapters/chapter06.rst [hunk 3]
```diff
@@ -0,0 +1,563 @@
...
+=========== ====== ========= ===============================================================================================
+
+Table 6-3-3: Network Virtualization parameter.
+
+server information
+~~~~~~~~~~~~~~~~~~
+
+Server information should be provided for installer, including full detail info. for each server, NIC mapping etc.
+
+.. _server-information-1:
```
Comment (CsatariGergely) [inline doc/ref_impl/cntt-ri/chapters/chapter06.rst:110]:
```suggestion
.. _ri1-ch6server-information:
```

File: doc/ref_impl/cntt-ri/chapters/chapter08.rst [hunk 4]
```diff
@@ -0,0 +1,638 @@
...
+4    2xE5-2699 64 GB  1 x 3 TB Sata 1 x 180 SSD 2         2
+5    2xE5-2699 64 GB  1 x 3 TB Sata 1 x 180 SSD 2         2
+==== ========= ====== ============= =========== ========= ==========
+
+Each server has all of 1GbE NICs connected to the same Extreme 480 1GbE switch, and all 10GbE NICs conneted to the
+same IZ1 switch as follows:
+
+.. image:: ../figures/ch07_pod10_switch_connectivity.png
+   :alt: Pod 10 Switch Connectivity
+
```
Comment (CsatariGergely) [inline doc/ref_impl/cntt-ri/chapters/chapter08.rst:37]: Please add caption

File: doc/ref_impl/cntt-ri/chapters/chapter08.rst [hunk 5]
```diff
@@ -0,0 +1,638 @@
...
+Requirements Gathering
+----------------------
+
+Requirements gathering processes and steps:
+
+1. RA Requirements Gathering process
+2. Infra Requirements and Selection Process
+
+.. _access--connectivity:
```
Comment (CsatariGergely) [inline doc/ref_impl/cntt-ri/chapters/chapter08.rst:46]:
```suggestion
.. _ri1-ch8-access--connectivity:
```

File: doc/ref_impl/cntt-ri/chapters/chapter08.rst [hunk 6]
```diff
@@ -0,0 +1,638 @@
...
+
+The descriptor files – platform, infrastructure and software descriptor files – as used in OPNFV, provides an installer-agnostic way of describing both the hardware and software infrastructure. The main consumer of these descriptor files is the openstack (OPNFV) installer. For example, Fuel consumes these descriptor files to create its own set of templates/manifests, which will be used for deployment. The below figure summarizes how these descriptor files are consumed by the installers.
+
+It is important for the generic descriptor files, apart from being generic, to be exhaustive enough to cover the parameters that are important and user-configurable. The importance of the parameters can be decided by various factors, and below we mention 3 of them:
+
+1. New Use-Cases: If the parameter(s) helps in creating new use-cases. Ex: Different networking scenarios.
+2. Feature: If the parameter(s) adds new feature to the installation. Ex: Security.
+3. Simplify: If the parameter(s) simplifies the auto-creation of custom manifests. Ex: Metadata.
+
+.. _opnfv-descriptor-files-1:
```
Comment (CsatariGergely) [inline doc/ref_impl/cntt-ri/chapters/chapter08.rst:130]:
```suggestion
.. _ri1-ch8-opnfv-descriptor-files:
```

File: doc/ref_impl/cntt-ri/chapters/chapter08.rst [hunk 7]
```diff
@@ -0,0 +1,638 @@
...
+-  mv site/intel-pod10 ../airship/site/
+
+Publishing
+''''''''''
+
+The process of publishing involves submitting the manifests to opnfv-airship gerrit repo. It is important that the site-specific manifest, along with certificates, is present in 'site' folder in opnfv-airship repository. This will be used for the deployment process and described in the subsequent section. An example record for pod-10 dashboard would be:
+
+-  A \| dashboard-airship.intel-pod10.opnfv.org \| 10.10.100.100
+
+.. _deployment-installer--install-steps:
```
Comment (CsatariGergely) [inline doc/ref_impl/cntt-ri/chapters/chapter08.rst:322]:
```suggestion
.. _ri1-ch8-deployment-installer--install-steps:
```

File: doc/ref_impl/cntt-ri/chapters/chapter08.rst [hunk 8]
```diff
@@ -0,0 +1,638 @@
...
+`currently CNTT compliant <https://build.opnfv.org/ci/view/functest/>`__.
+
+CICD Tool Chain (use of, process, and scripts)]
+-----------------------------------------------
+
+Placeholder to describe the CICD tool chain used in RI validations.
+
+Include flow diagram.
+
+.. _jenkins-setup--job-creation:
```
Comment (CsatariGergely) [inline doc/ref_impl/cntt-ri/chapters/chapter08.rst:628]:
```suggestion
.. _ri1-ch8-jenkins-setup--job-creation:
```

File: doc/ref_impl/cntt-ri/chapters/chapter09.rst [hunk 9]
```diff
@@ -0,0 +1,169 @@
...
+-  Average (packet) Drop Rate
+-  Average Latency
+-  Execution of different frame size, packet path, or chain count
+-  Testing single switch (VNF) packet path
+-  Testing chained-switch (VNF) packet paths
+-  Support SRIOV, OVS-DPDK, and VLAN configuration and/or options
+
+Note, when it comes to NFR-tooling, the goals are to provide light-weight solutions that can be packaged within a cookbook to accelerate lab validations, agnostic to the type of hardware in the environment. This will enable third party suppliers to achieve compliance expectations for the targeted architecture.
+
+.. _traffic-generators--nic:
```
Comment (CsatariGergely) [inline doc/ref_impl/cntt-ri/chapters/chapter09.rst:151]:
```suggestion
.. _ri1-ch9-traffic-generators--nic:
```

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: doc/ref_impl/cntt-ri/chapters/chapter03.rst [hunk 10]
```diff
@@ -0,0 +1,438 @@
...
+------------
+
+The VNF profile is used to describe every workload running on top of Cloud Infrastructure. The Cloud Infrastructure SW profile is used to describe the list of features provided by the hypervisor and host OS.
+
+The CNTT Reference Model will be referenced as **RM** to avoid long and duplicated reference titles.
+
+VNF profile
+-----------
+
+Any virtual network functions and/or cloud-native network functions must choose to run on one of the pre-defined of entries in Cloud Infrastructure Infrastructure Profiles Catalogue. As states in `RM: 4 Infrastructure Capabilities, Measurements and Catalogue: 4.2 Catalogue <https://cntt.readthedocs.io/en/latest/ref_model/chapters/chapter04.html#profiles-and-workload-flavours>`__ BMC_PRE_CONFIGURED, the entry uses the following naming convention.
```
[the remark this answers is not in the document]
Comment (pgoyal01) [reply doc/ref_impl/cntt-ri/chapters/chapter03.rst:16]: @CsatariGergely in your suggested change should the "." be also deleted?

File: doc/ref_impl/cntt-ri/chapters/chapter03.rst [hunk 11]
```diff
@@ -0,0 +1,438 @@
...
+
+``B/N <I opt> . <Flavour> . <S ext> . <A ext>``
+
+B/N is used to specify the instance type (Basic, Network Intensive), different instance types are associated with different acceleration extensions, network characteristics (`RM: Profiles Specifications & Capability Mapping <https://cntt.readthedocs.io/en/latest/ref_model/chapters/chapter04.html#profiles-specifications-capability-mapping>`__).
```
[the remark this answers is not in the document]
Comment (pgoyal01) [reply doc/ref_impl/cntt-ri/chapters/chapter03.rst:20]: same comment as before about the "."

File: doc/ref_impl/cntt-ri/chapters/chapter03.rst [hunk 12]
```diff
@@ -0,0 +1,438 @@
...
+..
+
+   Open Point 1: Does ONAP have some relevant spec or VNF declaration schema so that CNTT can re-use/revise to cover what we need ? Or define a new one ?
+
+   Open Point 2: What principles should be followed if some the pre-define VNF profile items does not match what actual requires ? How to adjust, "ceiling", "floor", "customise" ?
+
+Cloud Infrastructure SW profile
+-------------------------------
+
+`RM: 5.2 Cloud Infrastructure Software Profiles features and requirements <https://cntt.readthedocs.io/en/latest/ref_model/chapters/chapter05.html#cloud-infrastructure-software-profiles-features-and-requirements>`__ defines the Cloud Infrastructure software layer. The profile depicts the feature status of the
```
[the remark this answers is not in the document]
Comment (pgoyal01) [reply doc/ref_impl/cntt-ri/chapters/chapter03.rst:73]: delete the "."

File: doc/ref_impl/cntt-ri/chapters/chapter01.rst [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This document includes the requirement and deployment details of the the first reference implementation for CNTT. This first reference implementation follows the requirement and architecture design of the :doc:`OpenStack Based Reference Architecture <ref_arch/openstack/chapters/chapter01>`. This document will includes the detailed requirement of RI for CNTT, NFVi configurations according to the requirement of RA and each different flavor of VNF, Lab requirements for RI deployment and validation and installer requirement. This reference implementation will fully reveal the capabilities and features defined in CNTT RM and RA, and thus will be utilized as 'golden NFVi' in the Reference Compliance of CNTT.
+
+In order to promote quick deployment of RI and not re-invent wheels, the CNTT community works closely with OPNFV, utilizing and evolving OPNFV CI pipeline and installer automation and testing automation capabilities. Detailed collaboration and contribution to OPNFV will be concluded in the following sessions.
+
+Terminology
+~~~~~~~~~~~
+
+Terminology in this document will follow :doc:`common/glossary`.
```
[the remark this answers is not in the document]
Comment (gkunz) [reply doc/ref_impl/cntt-ri/chapters/chapter01.rst:17]: the doc directive makes use of the intersphinx mapping defined in the local conf. Actually, when applying the suggested change, the build fails for me.

File: doc/ref_impl/cntt-ri/chapters/chapter03.rst [hunk 14]
```diff
@@ -0,0 +1,438 @@
+.. _cloud-infrastructure--vnf-target-state--specification:
```
[the remark this answers is not in the document]
Comment (gkunz) [reply doc/ref_impl/cntt-ri/chapters/chapter03.rst:1]: I am sorry for leaving those in the rst files. They were created by pandoc, but I assumed we were going to use implicit section header references. So, I would rather like to remove all those explicit anchors.

File: doc/ref_impl/cntt-ri/chapters/chapter08.rst (hunk 6 above)
Comment (gkunz) [reply doc/ref_impl/cntt-ri/chapters/chapter08.rst:130]: see above: I'd like to remove this.

File: doc/ref_impl/cntt-ri/chapters/chapter08.rst (hunk 7 above)
Comment (gkunz) [reply doc/ref_impl/cntt-ri/chapters/chapter08.rst:322]: see above: I'd like to remove this.

File: doc/ref_impl/cntt-ri/chapters/chapter09.rst (hunk 9 above)
Comment (gkunz) [reply doc/ref_impl/cntt-ri/chapters/chapter09.rst:151]: see above: I'd like to remove this.

File: doc/ref_impl/cntt-ri/chapters/chapter08.rst (hunk 8 above)
Comment (gkunz) [reply doc/ref_impl/cntt-ri/chapters/chapter08.rst:628]: see above: I'd like to remove this.

File: doc/ref_impl/cntt-ri/chapters/chapter08.rst (hunk 5 above)
Comment (gkunz) [reply doc/ref_impl/cntt-ri/chapters/chapter08.rst:46]: see above: I'd like to remove this.

File: doc/ref_impl/cntt-ri/chapters/chapter03.rst (hunk 10 above)
[the remark this answers is not in the document]
Comment (gkunz) [reply doc/ref_impl/cntt-ri/chapters/chapter03.rst:16]: I agree with using the ref directive. The configuration for building RI1 individually was missing the link corrector and hence the links broke for me. I'll try to address that.

File: doc/ref_impl/cntt-ri/chapters/chapter01.rst (hunk 13 above)
[the remark this answers is not in the document]
Comment (CsatariGergely) [reply doc/ref_impl/cntt-ri/chapters/chapter01.rst:17]: 1. I was using `tox` from the top level directory. Should we agree to craft document specific `tox.ini`-s and build locally?
2. I was not aware that intersphinx can do this :)

File: doc/ref_impl/cntt-ri/chapters/chapter03.rst (hunk 14 above)
[the remark this answers is not in the document]
Comment (CsatariGergely) [reply doc/ref_impl/cntt-ri/chapters/chapter03.rst:1]: I'm okay with removing them, but I could not find a way to refer to the implicit section header references. Do you have a working example?

File: doc/ref_impl/cntt-ri/chapters/chapter03.rst (hunk 10 above)
[the remark this answers is not in the document]
Comment (CsatariGergely) [reply doc/ref_impl/cntt-ri/chapters/chapter03.rst:16]: > @CsatariGergely in your suggested change should the "." be also deleted?

Not from the filenames. (I assume you refer to the . in `chapter04.md`)

File: doc/ref_impl/cntt-ri/chapters/chapter03.rst (hunk 14 above)
[the remark this answers is not in the document]
Comment (gkunz) [reply doc/ref_impl/cntt-ri/chapters/chapter03.rst:1]: For intra-references using implicit header references, see https://github.com/cntt-n/CNTT/pull/2777/files#diff-774fcc11722dce98d0347ce9870d8c0959a06c61223d2ee4a1cc4516db7211b5R64

You have to use the autosectionlabel extention (see https://github.com/cntt-n/CNTT/pull/2777/files#diff-e59005d178179a798a25d32c0f18a66b77b4cd290462ac82d5594c23fb0ee0d6R15)

File: doc/ref_impl/cntt-ri/chapters/chapter03.rst (hunk 10 above)
[the remark this answers is not in the document]
Comment (gkunz) [reply doc/ref_impl/cntt-ri/chapters/chapter03.rst:16]: > @CsatariGergely in your suggested change should the "." be also deleted?

@pgoyal01 which "." are you referring to?

[the remark this answers is not in the document]
Comment (pgoyal01) [reply doc/ref_impl/cntt-ri/chapters/chapter03.rst:14]: This sentence needs to be corrected as pre-defined profiles or the catalogue doesn't exist. Also, should the section number "4.2" be deleted in the text.

Opened Issue #2780.

**Review by pgoyal01 — APPROVED** [review]

**Comment by pgoyal01** [comment]
There needs to be an issue to correct CNTT, OPNFV, etc.

**Review by walterkozlowski — APPROVED** [review]

**Review by walterkozlowski — APPROVED** [review]

**Comment by gkunz** [comment]
@CsatariGergely can you please re-view? In its current version, all checks are passing. I'd like to keep this PR concise and address the following in follow-up PRs: rename OPNFV to Anuket, replace links to rtd with intersphinx links.

**Pull request merged**